### ISSUE TITLE
Papertrader not working anymore in develop branch

### ISSUE THREAD

**Issue by bald123** [askmike/gekko#2341]
**Note: this is the technical bug tracker, please use other platforms for getting support and starting a (non technical) discussion. See the [getting help page](https://gekko.wizb.it/docs/introduction/getting-help.html) for details.**

**I'm submitting a ...**
[*] bug report
[ ] question about the decisions made in the repository

**Action taken** (what you did)
The whole last week I tried to run the latest code of the develop branch, but the papertrader seems not to be working anymore. 

> Exchange: kraken
> Currency: EUR
> Asset: XRP
> Type: papertrader
> 
> Strategy
> Name
> RSI_Bull_Bear_Adx_Stop
> Parameters
> 
> {
>     "SMA_long": 1000,
>     "SMA_short": 50,
>     "BULL_RSI": 10,
>     "BULL_RSI_high": 80,
>     "BULL_RSI_low": 60,
>     "BEAR_RSI": 15,
>     "BEAR_RSI_high": 50,
>     "BEAR_RSI_low": 20,
>     "BULL_MOD_high": 5,
>     "BULL_MOD_low": -5,
>     "BEAR_MOD_high": 15,
>     "BEAR_MOD_low": -5,
>     "ADX": 3,
>     "ADX_high": 70,
>     "ADX_low": 50,
>     "Stop_Loss_Percent": 50
> }

The result is error messages and a non-running Gekko, immediately after start:
- This is an archived Gekko, it is currently not running anymore.
- This is Gekko crashed with the following error: This Gekko instance encountered an error and can't continue

No error messages in the log files in the /logs directory. I tried it with other exchanges as well.

**Expected result** (what you hoped would happen)
A running papertrader.

**Actual result** (unexpected outcome)
Papertrader crashes immediately, no errors to be found in the logs.

**Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, etc)

**Comment by askmike** [comment]
Works great for me:

<img width="1552" alt="screen shot 2018-07-20 at 22 04 07" src="https://user-images.githubusercontent.com/969743/43006665-dcc2540a-8c68-11e8-8038-0153b1fbbf83.png">

Can you try with a default strategy that comes with Gekko?

**Comment by bald123** [comment]
Mike, I can't reproduce it anymore as well, I created a new built and tried several strategies and exchanges now with the papertrader. I think that I made a mistake somewhere, I won't be able to discover where it went wrong though, my apologies for that.

Now that I managed to get it working I'll start running a new test instance for the coming days with the latest develop code. Again, my apologies.

**Comment by bald123** [comment]
And now I can reproduce it, but not with the standard strategies. But with a lot of strategies I successfully used before, I encounter this error. Not sure what the problem is at the moment...

**Comment by bald123** [comment]
It turns out that when the following line is in the strategy file, it crashes:
this.addTulipIndicator('maSlow', 'sma', { optInTimePeriod: this.settings.smaLong });

When using the "tulip-adx" strategy, included in the default Gekko source, it crashes also!

**Comment by askmike** [comment]
I'm not sure if you are using the UI or the CLI to start, but I just some default configs for the two tulip methods: `tulip-macd` and `tulip-adx`, for some reason they were not included in the default config.

For me Gekko works fine using the sample config (that passes the correct config). Can you check if that also works for you?

**Comment by bald123** [comment]
Will check it asap and let you know! Im using the UI. What I did this morning was completely strip the strategy and adding line for line to see where it went wrong (could not find useful logs related to this issue). And it crashed on the addTulipIndicator... very strange.

**Comment by bald123** [comment]
I found the issue. In my deployment scripts, I injected the following line in Dockerfile:

RUN npm install mathjs convnetjs

This is necessary for some (external) indicators. It turns out that not injecting this line solved the problem. This worked before v0.6.0. I don't know anymore which indicators required these libraries, and I don't know if this is going to be another issue for me, but that's not related to Gekko then. I can't judge if this is a problem that should be solved.

With this issue, I noticed that when a strategy crashes in the init method, no errors are found in the log files (don't know if this is by design or not).
